packages/at_secondary_server/lib/src/notification/resource_manager.dart

```diff
@@ -235,10 +235,11 @@ class ResourceManager {
         commandBody =
             '${AtConstants.sharedKeyEncrypted}:${atNotification.atMetadata!.sharedKeyEnc}:$commandBody';
       }
-      if (atNotification.atMetadata!.isEncrypted != null &&
-          atNotification.atMetadata!.isEncrypted == true) {
-        commandBody = '${AtConstants.isEncrypted}:true:$commandBody';
-      }
+
+      String? isEncryptedStr =
+          (atNotification.atMetadata!.isEncrypted ?? false) ? 'true' : 'false';
+      commandBody = '${AtConstants.isEncrypted}:$isEncryptedStr:$commandBody';
+
```

Comment:
The above change is to ensure that the isEncrypted *as stored* in the notification's metadata is always passed on to the next atServer